client/webserver/site/src/css/main.scss

```diff
@@ -77,7 +77,7 @@ a.logo {
 div.main {
   display: flex;
   flex-grow: 1;
-  min-height: 0;
+  min-height: min-content;
```

Comment:
I can't seem to figure out why I did this. Hmmm. But this seems to be the source of the unscrollable markets view.